### PR TITLE
add docs to onnx APIs

### LINKED ISSUE #14698: torch.onnx.export is missing documentation on the docs page
## 📚 Documentation

`torch.onnx.export` shows no documentation on the docs [page](https://pytorch.org/docs/master/onnx.html#functions), while it has documentation in the [source](https://github.com/pytorch/pytorch/blob/master/torch/onnx/utils.py). It would be useful to have the documentation on the docs page.

### PR DESCRIPTION
**Pull request by zhangguanheng66**
Add docs to onnx APIs, including
  - export
  - export_to_pretty_string
  - is_in_onnx_export

Fix https://github.com/pytorch/pytorch/issues/14698

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

File: torch/onnx/__init__.py [hunk 1]
```diff
@@ -27,9 +27,109 @@ def _export(*args, **kwargs):
     return result
 
 
-def export(*args, **kwargs):
+def export(model, args, f, export_params=True, verbose=False, training=False,
```
Comment (houseroad) [inline torch/onnx/__init__.py]: please don't duplicate the doc. If we move the doc here, please delete the doc in utils.py.

**Review by houseroad — COMMENTED** [review]
Also cc: @spandantiwari who will put more effort into the improvement of the doc for onnx.

File: torch/onnx/__init__.py (hunk 1 above)
Comment (zhangguanheng66) [reply torch/onnx/__init__.py]: Delete the duplicate doc in onnx.utils.py

**Comment by spandantiwari** [comment]
@zhangguanheng66 @houseroad - Noted.

**Pull request closed without merging**